test/cypress/integration/widgets/blueprints_spec.ts

```diff
@@ -558,6 +567,32 @@ describe('Blueprints widget', () => {
             });
         });
 
+        it('enable to enter non-existing secret value', () => {
```

Comment:
Shouldn't it be:
```suggestion
        it('enable to enter non-existing secret key', () => {
```
?

Reply:
Oh yeah, I've got myself confused about that one
Thanks for pointing that out! :)